src/hotspot/share/c1/c1_LIRGenerator.cpp

```diff
@@ -1234,7 +1234,6 @@ void LIRGenerator::do_isInstance(Intrinsic* x) {
 void LIRGenerator::load_klass(LIR_Opr obj, LIR_Opr klass, CodeEmitInfo* null_check_info) {
   CodeStub* slow_path = new LoadKlassStub(obj, klass);
   __ load_klass(obj, klass, null_check_info, slow_path);
-  __ branch_destination(slow_path->continuation());
```

Comment:
Why this one is removed? I'd expect stub's `jmp(continuation)` to get here?

Reply:
It has actually been bound twice (to the same effective location), the other one is in c1_LIRAssembler_XXX.cpp:

https://github.com/rkennke/lilliput/blob/ab8ed6331fb4eb2c5eb0e13499178f8ef67dc0f6/src/hotspot/cpu/x86/c1_LIRAssembler_x86.cpp#L3546

It is important now to do it in C1_LIRAssembler, because we want the decode_klass() after the common path.